rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -294,25 +308,100 @@ namespace rsx
 					{
 						// Actually uses the modulo operator
 						_min_index = 0;
-						_max_index = attrib.frequency - 1;
+						_max_index = std::max<u32>(_max_index, attrib.frequency - 1);
```

Comment:
This is another bugfix, it should do max not assign,

Reply:
No, it shouldn't. This is a memory range to read, we should never read above the modulo wraparound.

Reply:
Ah, nvm, its correct because of the attrib loop.